heartexlabs/label-studio
# docs: Fix details tags

Fix https://github.com/HumanSignal/label-studio/issues/4563

Since we switched to yarn, package.json and yarn.lock files needed to be updated to fix the tags

### How to QA

The tags should work correctly:

https://deploy-preview-4564--heartex-docs.netlify.app/guide/predictions.html#Import-bbox-and-choice-pre-annotations-for-images

https://deploy-preview-4564--label-studio-docs-new-theme.netlify.app/guide/predictions.html#Import-bbox-and-choice-pre-annotations-for-images
